packages/react-composition/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./Context";
+// export type { HigherOrderComponent } from "./Context";
+export * from "./Compose";
+// export type { ComposeProps, ComposableFC } from "./Compose";
```

Comment:
🙈

Reply:
Ah good catch, forgot to double-check these.